lib/jira/commands/checkout.rb

```diff
@@ -0,0 +1,88 @@
+module Jira
+  class CLI < Thor
+
+    desc "checkout", "Checks out a ticket from JIRA in the git branch"
+    method_option :remote, aliases: "-r", type: :string, default: nil, lazy_default: "", banner: "REMOTE"
```

Comment:
would prefer if we used more explicit method_options - `--remote` instead of `-r`, but it's fine to leave as is for now until command arguments are standardized